source/agora/test/Ledger.d

```diff
@@ -53,10 +53,9 @@ unittest
         // send it to one node
         txs.each!(tx => node_1.putTransaction(tx));
 
-        Thread.sleep(50.msecs);  // await gossip and block creation
-
         nodes.enumerate.each!((idx, node) =>
-            assert(node.getBlockHeight() == block_idx + 1,
+            retryFor(node.getBlockHeight() == block_idx + 1,
```

Comment:
Note: If you still want to use UFCS, you can just remove the `assert` and leave the parenthesis.
But I'm not sure we should go down this road 🤔

Reply:
Yeah I know, I just found it a bit weird that way.